app/migrations/20250306124233-adds_project_table.cjs

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("Project", {
+      project_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      organization_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: "Organization",
+          key: "organization_id",
+        },
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+      },
+      city_count_limit: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        defaultValue: 0,
+      },
+      name: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      description: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
```

Comment:
Attributes in table should be `snake_case` and remapped in the model, see above.